javascript/javascript1/week4/hyf-homework-week4/voice.js

```diff
@@ -0,0 +1,96 @@
+console.log('Voice Assistance')
+
+let nameAssistance=[]
+ let todoList=[]
+ function getReply(command) {   
+
+    if(command.includes('What is my name')){
+        if (nameAssistance != "")
```

Comment:
for checking if array is empty or not you can check the length like 
`if (nameAssistance.length)`